patches/theide.patch

```diff
@@ -0,0 +1,403 @@
+diff --git a/uppsrc/Core/config.h b/uppsrc/Core/config.h
```

Comment:
https://github.com/ultimatepp/ultimatepp/pull/177 - will be merged to master branch once org.ultimatepp.TheIDE will be available on Flathub.